code/game/objects/items/reagent_containers/food/drinks.dm

```diff
@@ -313,6 +314,13 @@
 	reagents.add_reagent("water", 59)
 	reagents.add_reagent("hooch", 1)
 
+/obj/item/reagent_container/food/drinks/flask/weylandyutani
```

Comment:
likewise

Reply:
Same as Classic Cola, more meant for map makers and admins. One can be added to the liaison's office potentially, but I did not want to edit the map in this PR.